Content.Shared/Medical/DefibrillatorComponent.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Content.Shared.Damage;
+using Content.Shared.DoAfter;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Medical;
+
+/// <summary>
+/// This is used for defibrillators; a machine that shocks a dead
+/// person back into the world of the living.
+/// </summary>
+[RegisterComponent]
```

Comment:
```suggestion
[RegisterComponent, NetworkedComponent]
```

also Access.

Reply:
no access because server-only system but otherwise yeah.
(rotting, powercells, and various other shit make this a PITA to predict properly. spare me)